### DISCUSSION TITLE
Avoid autoplaying the next chapter

### DISCUSSION THREAD

**Discussion by InterferencePattern** [advplyr/audiobookshelf#2051] [Ideas]
Apologies if this is already addressed, but I didn't see any other discussions about auto-playback.

I have a particular book where I don't want to auto-play the next chapter- it would be better if it would stop playing and wait for me to hit Stream again to advance.

I don't see any such option in the Android app, but I think this would be useful for particular types of books (e.g. guided meditation sessions)

**Comment by nichwall** [comment]
You can use the sleep timer in the Android app and select end of chapter. You can also disable fade out in the Android settings
![Screenshot_20230902-161143.png](https://github.com/advplyr/audiobookshelf/assets/5686638/149667f6-f895-47c3-8fcb-8ffc8684db64)

**Reply by InterferencePattern** [reply]
The sleep timer works, though it'd be nice to be able to assign this consistently on a per-book or global basis.

That being said, this is probably a niche request, so I think the existing solution is "close enough".

**Reply by nichwall** [reply]
Yeah, that makes sense. It won't make it a per-book basis, but you could also set it to always be on by using "Auto Sleep Timer" in the settings (under hamburger menu) and getting creative with the timing so it's "always on".
